app/controllers/home_controller.rb

```diff
@@ -17,6 +17,7 @@ def index
                             .lessons_this_month
     @past_lessons = Lesson.for_school_id(current_school.id).past_lessons
     @future_lessons = Lesson.for_school_id(current_school.id).future_lessons
+    @job_posts = JobPost.for_school_id(current_school.id).active
```

Comment:
Without a scope, this could be `JobPost.where(school: current_school)` which is preferable to me than the extra, not particularly useful scope. Or `current_school.job_posts` if you set up the has_many side.